src/internal/puck/ui/switch.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import * as SwitchPrimitives from "@radix-ui/react-switch";
+import { cn } from "../../utils/cn.ts";
+
+const Switch = React.forwardRef<
+  React.ElementRef<typeof SwitchPrimitives.Root>,
+  React.ComponentPropsWithoutRef<typeof SwitchPrimitives.Root>
+>(({ className, ...props }, ref) => (
+  <SwitchPrimitives.Root
+    className={cn(
+      "self-center peer inline-flex h-5 w-9 shrink-0 cursor-pointer items-center rounded-full border-2 border-transparent transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 focus-visible:ring-offset-background disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=unchecked]:bg-input",
```

Comment:
Tailwind classes in the ve library are prefixed with "ve-" to isolate their styles. So all of these class names have to be prefixed like "ve-items-center". This is true wherever you're using tailwind classes in this lib.

Reply:
Gotcha. I think there are some libraries where u can have "ve-" automatically added to all the tailwind classes too on build or smth. I'll fix tho!